src/cryfs-cli/Cli.cpp

```diff
@@ -324,10 +324,10 @@ namespace cryfs_cli {
     }
 
 	void Cli::_sanityChecks(const ProgramOptions &options) {
-		_checkDirAccessible(bf::absolute(options.baseDir()), "base directory", ErrorCode::InaccessibleBaseDir);
+		_checkDirAccessible(bf::absolute(options.baseDir()), "base directory", options.createMissingMountpoint(), ErrorCode::InaccessibleBaseDir);
```

Comment:
It's a bit confusing that a `--create-missing-mountpoint` flag also automatically creates the missing base directory if it doesn't exist. We should probably have a separate flag for the base dir.